modules/widgets/flickr/widget.php

```diff
@@ -0,0 +1,33 @@
+<!-- Start of Flickr Widget -->
+<table
+	border="0"
+	cellpadding="0"
+	cellspacing="0"
+	class="flickr-size-<?php echo esc_attr( $instance['flickr_image_size'] ); ?>"
+	id="flickr_badge_uber_wrapper"
+>
+	<tr>
+		<td>
+			<table border="0" cellpadding="0" cellspacing="10" id="flickr_badge_wrapper">
+				<tr>
+					<td align="center">
+						<?php foreach ( $photos as $key => $photo ) { ?>
+							<a href="<?php echo $photo['link']; ?>">
+								<img
+									alt="<?php echo $photo['title']; ?>"
+									border="0"
+									src="<?php echo $photo['src']; ?>"
```

Comment:
I'm thinking we could apply Photon to those images if the module is enabled, like we do for the Image and the Text widget.